applications/stream-applications-core/pom.xml

```diff
@@ -18,7 +18,7 @@
     <properties>
         <stream-apps-core.version>5.0.1-SNAPSHOT</stream-apps-core.version>
         <java-functions.version>5.0.1-SNAPSHOT</java-functions.version>
-		<prometheus-rsocket.version>1.5.2</prometheus-rsocket.version>
+        <prometheus-rsocket.version>2.0.0-M1</prometheus-rsocket.version>
```

Comment:
This would be possible if we would plan `5.1` version for this project.
Or `1.x` generation for that dependency is not compatible with current Micrometer?

Reply:
The `1.x` generation is based on Spring Boot 2.7.x. The `2.x` generation is based on Spring Boot 3.3.x. I should have pointed this out w/ a self-review comment. Thanks for bringing it up.

Reply:
I know that was unfortunate that we had missed this before 5.0 GA, but to go on with such an upgrade I'd like to see a GA version as well. This might be just a dependency of our uber artifact (or Docker image), but still would be great to have it in Maven Central for consistency and so on.
Do you have any dates when this Prometheus Rsocket library is going 2. GA?

Reply:
The GA is end of September last time I checked w/ Micrometer team. We are also using the M1 in Dataflow until that time.

Reply:
This would stay in the snapshot until the GA is available.